my-app/src/components/ContentList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```

Comment:
in the current version of react that you're using; you don't need to import `React`.